org.biblemulti.thelife.yml

```diff
@@ -0,0 +1,36 @@
+app-id: org.biblemulti.thelife
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: biblemultithelife
+separate-locales: false
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: libkeybinder
+    buildsystem: autotools
+    config-opts:
+       - --disable-introspection
+    sources:
+       - type: archive
+         url: https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
+         sha256: e6e3de4e1f3b201814a956ab8f16dfc8a262db1937ff1eee4d855365398c6020
+  - name: biblemultithelife
+    buildsystem: simple
+    build-commands:
+       - mkdir -p /app
+       - tar -xf biblemultithelife.tar.gz -C /app
+       - mkdir -p /app/bin
+       - ln -s /app/biblemultithelife /app/bin/biblemultithelife
+       - ln -s /usr/lib/x86_64-linux-gnu/libsqlite3.so.0 /app/lib/libsqlite3.so
+    sources:
+      - type: file
+        url: https://gitlab.com/-/project/17175284/uploads/eac870e763585b0b2518691632fdda7d/biblemultithelife.tar.gz
```

Comment:
why is this not versioned? only pinned to a commit... are you not doing releases?

Reply:
it's a release: note-1.27.4 
I can't tag as I want as F-Droid is also checking the tags.